.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "typescript.tsdk": "/Users/kswenson/Development/cc-dev/lara-debugging-plugin/node_modules/typescript/lib"
```

Comment:
Should this file be here with your local path in it?

Reply:
The goal was to tell VSCode to use the workspace version of TypeScript rather than the VSCode default version. Since it's a project-local setting I'd assumed it would use a project-local path, but apparently not so I'll just remove it.